
ERROR: The SAMPLEDATA dbconsinstency has fail:
modules/org.openbravo.retail.sampledata:
diff --git a/referencedata/sampledata/The_White_Valley_Group/C_BPARTNER.xml b/referencedata/sampledata/The_White_Valley_Group/C_BPARTNER.xml
index d7a2447..e6615b3 100644
--- a/referencedata/sampledata/The_White_Valley_Group/C_BPARTNER.xml
+++ b/referencedata/sampledata/The_White_Valley_Group/C_BPARTNER.xml
@@ -47,6 +47,7 @@
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -101,6 +102,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -156,6 +158,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -203,6 +206,7 @@
   <PO_GOODS_BLOCKING><![CDATA[N]]></PO_GOODS_BLOCKING>
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -256,6 +260,7 @@
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -310,6 +315,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -359,6 +365,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -408,6 +415,7 @@
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -460,6 +468,7 @@
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -515,6 +524,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -568,6 +578,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -615,6 +626,7 @@
   <PO_GOODS_BLOCKING><![CDATA[N]]></PO_GOODS_BLOCKING>
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -670,6 +682,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -724,6 +737,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -778,6 +792,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -827,6 +842,7 @@
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -883,6 +899,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -936,6 +953,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -991,6 +1009,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -1045,6 +1064,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -1099,6 +1119,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -1148,6 +1169,7 @@
   <ISCASHVAT><![CDATA[N]]></ISCASHVAT>
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -1203,6 +1225,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 <C_BPARTNER>
@@ -1252,6 +1275,7 @@
   <UPDATE_CURRENCY><![CDATA[N]]></UPDATE_CURRENCY>
   <BP_CURRENCY_ID><![CDATA[102]]></BP_CURRENCY_ID>
   <IS_CUSTOMER_CONSENT><![CDATA[N]]></IS_CUSTOMER_CONSENT>
+  <IS_CUSTOMER_CONSENT_PAPFORMAT><![CDATA[N]]></IS_CUSTOMER_CONSENT_PAPFORMAT>
 </C_BPARTNER>
 
 </data>

